confiture-web-app/src/components/SiteHeader.vue

```diff
@@ -1,59 +1,79 @@
 <script setup lang="ts">
 import { computed } from "vue";
-import { RouteLocationRaw } from "vue-router";
+import { useRoute, RouteLocationRaw } from "vue-router";
 
 import { useAuditStore, useReportStore } from "../store";
 
 const reportStore = useReportStore();
 const auditStore = useAuditStore();
 
-const logoLink = computed(() => {
-  if (reportStore.data) {
-    return {
-      route: {
-        name: "report",
-        params: { uniqueId: reportStore.data.consultUniqueId },
-      },
-      title: "Rapport d’audit - Ara",
-    };
-  }
-  return {
-    route: { name: "home" },
-    title: "Accueil - Ara",
-  };
+const currentRoute = useRoute();
+const matchedRoutesNames = computed(() => {
+  return currentRoute.matched.map((r) => r.name);
 });
 
-const homeLocation = { label: "Accueil", to: { name: "home" } };
-const helpLocation = { label: "Aide", to: { name: "help" } };
-const resourcesLocation = { label: "Ressources", to: { name: "resources" } };
-
-const menuItems = computed<Array<{ to: RouteLocationRaw; label: string }>>(
-  () => {
-    if (reportStore.data) {
-      const reportLocation = {
-        to: {
-          name: "report",
-          params: { uniqueId: reportStore.data.consultUniqueId },
-        },
-        label: "Rapport d’audit",
-      };
-      return [reportLocation, resourcesLocation, helpLocation];
+/**
+ * Determine if the navigation link should has be highlighted.
+ */
+function isCurrent(to: RouteLocationRaw, match?: string) {
```

Comment:
Nom de la fonction pas représentatif

```suggestion
function getAriaCurrentValue(to: RouteLocationRaw, match?: string) {
```